README.md

```diff
@@ -22,11 +22,9 @@
 - Please read _all_ the links included:
 - Take a look at [our **Guidebook**](https://bwtc-handbook.notion.site/BWTC-Handbook-71256770d53f46439aecd989b07d1533) for all your (mostly) non-code related needs!
 
-### GitHub
+#### GitHub
```

Comment:
I almost feel like a whole other header for this little bit of information isn't neccesary. We could just add it under the heading for "New to BitWise"?